assets/dev/js/editor/controls/media.js

```diff
@@ -243,6 +251,54 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 	onBeforeDestroy() {
 		this.$el.remove();
 	},
+
+	async onMediaImageSize() {
+		this.ui.mediaImageSize.css( 'color', 'inherit' );
+		const previewPlaceHolder = this.$el.find( '.elementor-control-media__preview' );
```

Comment:
You should have this element under `this.ui.mediaImage`

Reply:
Done